question/export_form.php

```diff
@@ -34,6 +34,7 @@
  *
  * @copyright  2007 Jamie Pratt me@jamiep.org
  * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ * @deprecated since Moodle 4.0 MDL-70329 - Please use \core_question\bank\export\export_form.php instead.
  */
 class question_export_form extends moodleform {
 
```

Comment:
maybe also a deprecation msg in the definition so that anyone using that gets a debug msg